convert2rhel/actions/conversion/pkg_manager_config.py

```diff
@@ -0,0 +1,38 @@
+# Copyright(C) 2024 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+import logging
+
+from convert2rhel import actions, redhatrelease
+
+
+logger = logging.getLogger(__name__)
+
+
+class ConfigurePkgManager(actions.Action):
+    id = "CONFIGURE_PKG_MANAGER"
+
```

Comment:
In the past times, the patching was executed after the [main transaction](https://github.com/oamg/convert2rhel/blob/v1.6.1/convert2rhel/main.py#L297). I've tried that, and it seems it's not a problem, having the `yum.conf` patched before the transaction.

But maybe just for staying safe - shouldn't there be the transaction as dependency?

Reply:
Otherwise, the PR looks good!

Reply:
Sure let me add that